cmd/gapit/profile.go

```diff
@@ -16,31 +16,53 @@ package main
 
 import (
 	"context"
-	"encoding/csv"
 	"flag"
-	"fmt"
-	"io"
+	"io/ioutil"
 	"os"
 	"path/filepath"
-	"strings"
 
+	perfetto_pb "perfetto/config"
+
+	"github.com/golang/protobuf/proto"
 	"github.com/google/gapid/core/app"
 	"github.com/google/gapid/core/log"
 	"github.com/google/gapid/gapis/service"
-	"github.com/google/gapid/gapis/service/path"
 )
 
-type profileVerb struct{ GetTimestampsFlags }
+const (
+	bufferSizeKb                            = uint32(131072)
```

Comment:
can we have some comments about those numbers?

Reply:
Sure, I shamelessly took them from gapis/trace/android/trace.go  It's possible that we should use different values in any case.